### ISSUE TITLE
Implementation proposal

### ISSUE THREAD

**Issue by anisiomqs** [alexsandrofmendes/calculos-textil#1]
[Here, in repl.it,](https://repl.it/J0W6/31) I've implemented a proposal to achieve what you need. I'm using a key-value pair structure to represent the conversion table (named `table` in `converter.js`). For now, only the first line of the table is there -- representing the conversion from "Ne" to other units.

I hope the test suite (`converter-test.js`) is clear enough -- it iterates through the target units checking if the conversion outputs the expected value.

I also decided to to implement the conversion as the product of two factors. When division is necessary, you can invert one of the factors (`1/factor`).

Suggestions are welcome.

**Comment by alexsandrofmendes** [comment]
I like... do you propose a form simple, building line per line... but we gonna build 36 formulas, right?..

In output, we gonna insert o Math.jax? Him perform operation and print the formula or we should create a code line for make the calculation and print the result?

**Comment by anisiomqs** [comment]
Yes. We need to represent the whole conversion table in our `table` object. 

We could build upon this implementation to comprehend a string representing the formulas, which could be used later to be parsed and presented using MathJax. 

However, in my opinion, this should be taken care after the basic functionality is available.
